pkg/core/exec_unit.go

```diff
@@ -74,8 +74,6 @@ var (
 	ExecutableTypeGolang = ExecutableType("Golang")
 )
 
-func (unit *ExecutionUnit) Type() string { return unit.ExecType }
```

Comment:
Should `ExecType` (the field) also be removed?

Reply:
good catch, will go remove